.travis.yml

```diff
@@ -1,13 +1,13 @@
 language: python
 required: sudo
 python:
-    "2.7"
+    - "2.7"
+    - "3.2"
 virtualenv:
     system_site_packages: true
 install:
-  - sudo apt-get install python-opencv
+  - sudo -H python -m pip install opencv-python pillow
```

Comment:
So OpenCV 2 tests for Python 2 and OpenCV 3 tests for Python 3? This whole Python 3 testing is already quite difficult, doing OpenCV is even more difficult... I could do an if-statement for the Python version and install OpenCV 2 for Python 2, and 3 for 3.